.semaphore/semaphore.yml

```diff
@@ -80,6 +64,50 @@ blocks:
         - name: Swift tests
           commands:
             - bundle exec fastlane test
+  - name: Appium UI Tests
+    dependencies:
+      - Appium UI Tests(Initialization)
+    task:
+      env_vars:
+        - name: LANG
+          value: en_US.UTF-8
+        - name: SEMAPHORE_GIT_DIR
+          value: /Users/semaphore/git/flowcrypt-ios
+      prologue:
+        commands:
+          - checkout && cd ~/git/flowcrypt-ios/ && cache restore && make dependencies
+          - mv ~/appium-env ~/git/flowcrypt-ios/appium/.env
+          - sem-version node 16 && cache restore appium-npm && cd ./appium && npm i && cd .. && cache store appium-npm appium/node_modules
+          - cd appium
+          - artifact pull workflow FlowCrypt.app
+      epilogue:
+        always:
+          commands:
+            - artifact push workflow ~/git/flowcrypt-ios/appium/tmp
+            - test-results publish ~/git/flowcrypt-ios/appium/tmp/test-results
+        on_fail:
+          commands:
+            - artifact push workflow ~/Library/Logs/DiagnosticReports/FlowCrypt_*
+            - artifact push workflow ~/git/flowcrypt-ios/appium/video/*.mp4
+      jobs:
+        - name: Run mock tests
+          commands:
+            - npm run-script test.mock.all
+        - name: Run Live tests
+          commands:
+            - 'wget https://flowcrypt.s3.eu-central-1.amazonaws.com/release/flowcrypt-ios-old-version-for-ci-storage-compatibility-2022-05-09.zip -P ~/git/flowcrypt-ios/appium'
+            - unzip flowcrypt-ios-*.zip
+            - npm run-script test.live.all
+      secrets:
+        - name: flowcrypt-ios-ci-secrets
+  - name: Delete Generated Build
+    dependencies:
+      - Appium UI Tests
+    task:
+      jobs:
+        - name: Delete FlowCrypt.app
+          commands:
+            - artifact yank workflow FlowCrypt.app
```

Comment:
You could try use `cache store `FlowCrypt-$SEMAPHORE_GIT_SHA.app FlowCrypt.app`  and then `cache restore `FlowCrypt-$SEMAPHORE_GIT_SHA.app`

The advantage is that you don't have to later delete it, oldest cache entries get automatically deleted as you write new ones.

Reply:
Got it